### PR TITLE
Create ecip-randomx-revival.md

### PR DESCRIPTION
**Pull request by gitr0n1n**
+ resubmit proposal to return to WIP or Draft status per requirment in: https://github.com/ethereumclassic/ECIPs/pull/359#issuecomment-687546746

### CODE REVIEW

File: _specs/ecip-randox-revival.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+lang: en
+ecip: xxxx
```
Comment (realcodywburns) [inline _specs/ecip-randox-revival.md:3]: please update to ecip 1100

File: _specs/ecip-randox-revival.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+## Donations
+
+If you'd like to use RandomX, please consider donating to help cover the development cost of the algorithm.
+
+Author's XMR address:
+```
+845xHUh5GvfHwc2R8DVJCE7BT2sd4YEcmjG8GNSdmeNsP5DTEjXd1CNgxTcjHjiFuthRHAoVEJjM7GyKzQKLJtbd56xbh7V
+```
+Total donations received: ~3.86 XMR (as of 30th August 2019). Thanks to all contributors.
```
Comment (realcodywburns) [inline _specs/ecip-randox-revival.md:194]: please add license footer

**Review by realcodywburns — CHANGES_REQUESTED** [review]

**Comment by q9f** [comment]
You already managed to move ECIP 1093 back to draft. This PR does not add any value.

Closing as duplicate of 1093.

**Comment by gitr0n1n** [comment]
> You already managed to move ECIP 1093 back to draft. This PR does not add any value.
> 
> Closing as duplicate of 1093.

Thanks @q9f .

**Pull request closed without merging**